xlineapi/src/lib.rs

```diff
@@ -455,11 +455,6 @@ impl RequestWrapper {
     pub fn is_kv_request(&self) -> bool {
         self.backend() == RequestBackend::Kv
     }
-
-    /// Check if this request is a lease request
```

Comment:
Why do we remove this method?

Reply:
Because we only check lease request's lease id before, so we need this method to check whether it is a Lease request, but now we will check all the request‘s lease ID, not just the lease request.